packaging/torchdata/bld.bat

```diff
@@ -6,6 +6,8 @@
 
 @echo off
 
+git config --system core.longpaths true
+
```

Comment:
This is required because a few submodules that AWSSDK used have long name for windows system